cmd/migrate/db/operations.go

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+package db
```

Comment:
```suggestion
package mariadb
```
Suggestion to rename this file to
`cmd/migrate/mariadb/operations.go` 🤔 ?